### PR TITLE
add subscript & superscript

### PR DESCRIPTION
**Pull request by TorstenDittmann**
- add sub- and superscript tags
- add tests

### CODE REVIEW

File: spec/selection.spec.js [hunk 1]
```diff
@@ -218,6 +218,68 @@ describe('Selection', function () {
...
     })
 
+    describe('superscript:', function () {
+      beforeEach(function () {
+        this.oldSuperscriptMarkup = config.superscriptMarkup
+        config.superscriptMarkup = {
+          type: 'tag',
+          name: 'sup',
+          attribs: {
+            'class': 'bar'
```
Comment (jenstroeger) [inline spec/selection.spec.js:228]: Looks like debug residue?

File: spec/selection.spec.js [hunk 2]
```diff
@@ -218,6 +218,68 @@ describe('Selection', function () {
...
+    })
+
+    describe('subscript:', function () {
+      beforeEach(function () {
+        this.oldSubscriptMarkup = config.subscriptMarkup
+        config.subscriptMarkup = {
+          type: 'tag',
+          name: 'sub',
+          attribs: {
+            'class': 'bar'
```
Comment (jenstroeger) [inline spec/selection.spec.js:259]: Here too, this looks like debug residue?

**Comment by jenstroeger** [comment]
@TorstenDittmann, any chance you could add two buttons to the [example tooltip](https://github.com/livingdocsIO/editable.js/blob/7dcf471a50461a447a8357d0c8731a1778e8919d/examples/index.js#L47) just for demo purposes of this feature?

**Comment by TorstenDittmann** [comment]
This is actually not necessary, I realized that afterwards. 

You can just use `forceWrap`, subscript and superscript might be too explicit.

**Comment by jenstroeger** [comment]
> This is actually not necessary […]

How so? [`forceWrap()`](https://github.com/livingdocsIO/editable.js/blob/7dcf471a50461a447a8357d0c8731a1778e8919d/src/selection.js#L238-L242) and [`toggle()`](https://github.com/livingdocsIO/editable.js/blob/7dcf471a50461a447a8357d0c8731a1778e8919d/src/selection.js#L116-L120) help to take care of adding/removing elements to/from selected content, whereas your change adds support for [super](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/sup)/[subscript](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/sub) elements. You mean, a user might as well add these elements using `forceWrap()` and `toggle()` herself, no need for your helper helper functions?

**Comment by TorstenDittmann** [comment]
> > This is actually not necessary […]
> 
> How so? [`forceWrap()`](https://github.com/livingdocsIO/editable.js/blob/7dcf471a50461a447a8357d0c8731a1778e8919d/src/selection.js#L238-L242) and [`toggle()`](https://github.com/livingdocsIO/editable.js/blob/7dcf471a50461a447a8357d0c8731a1778e8919d/src/selection.js#L116-L120) help to take care of adding/removing elements to/from selected content, whereas your change adds support for [super](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/sup)/[subscript](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/sub) elements. You mean, a user might as well add these elements using `forceWrap()` and `toggle()` herself, no need for your helper helper functions?

Yeah sorry, thats what I meant 👍

**Comment by jenstroeger** [comment]
Ok, gotcha. I’m a fan of simple helper functions so I thought your contribution is useful. But let’s leave that decision to @peyerluk… 😉

**Pull request closed without merging**